### DISCUSSION TITLE
Sign APKs in GitHub releases

### DISCUSSION THREAD

**Discussion by hardcore-sushi** [Emurgo/yoroi-mobile#1881] [Ideas]
Hi !
For obvious reasons, I don't want to use Google Play to install Yoroi. However, I don't trust neither GitHub nor the datacenter where it's hosted. APKs that users download might not be the same as the ones you uploaded.
To prevent any modification by GitHub or any other intermediaries, you could sign the APKs you upload, for example with PGP, like Daedalus does. You could publish your public key on public keyservers as well as on your website, as it secured with TLS.
Thank you very much for your understanding.

**Comment by zuzunker** [comment]
Related:
- https://github.com/Emurgo/yoroi-mobile/issues/707
